financialaid/models_test.py

```diff
@@ -90,3 +88,40 @@ def test_save_and_log(self):  # pylint: disable=no-self-use
                 self.assertAlmostEqual(value, original_after_json[field])
             else:
                 assert value == original_after_json[field]
+
+    def test_to_dict(self):
+        """
+        Tests the to_dict() function on FinancialAid
+        """
+        financial_aid = FinancialAidFactory.create(date_documents_sent=datetime.datetime.now())
+        financial_aid_dict = financial_aid.to_dict()
+        audit_key_list = [
+            "user",
+            "tier_program",
+            "status",
+            "original_currency",
+            "country_of_income",
+            "date_exchange_rate",
+            "date_documents_sent",
+            "income_usd",
+            "original_income"
+        ]
+        assert set(financial_aid_dict.keys()).issuperset(set(audit_key_list))
```

Comment:
why `issuperset`? If someone adds more fields to `FinancialAid`, this test should probably fail and force the engineer to add the extra field here.